packages/ui/src/page/dashboard/Dashboard.jsx

```diff
@@ -63,7 +64,14 @@ function Dashboard() {
   };
 
   if (loading) {
-    return <div>loading..</div>;
+    return (
+      <div
+        data-testid="loading-spinner"
+        className={styles["spinner-container"]}
```

Comment:
nit: you can name `spinner-container` class as Pascal case that way you can do `styles.SpinnerContainer`

Reply:
ok